hello-typescript/src/reverse.ts

```diff
@@ -0,0 +1,35 @@
+// Reverse any array
+export function reverseArray<T>(array: T[]): T[] {
+  const reverseAnything: T[] = [];
+  for (let i = 0; i < array.length; i++) {
+    reverseAnything.push(array[array.length - 1 - i]);
+  }
```

Comment:
بهتر بود حلقه رو از آخر به اول پیمایش کنید که خواناتر باشه

```suggestion
  for (let i = array.length - 1; i >= 0; i--) {
    reverseAnything.push(array[i]);
  }
```